test/gtest/cba_infer.cpp

```diff
@@ -131,6 +148,25 @@ TEST_P(ConvBiasActivInferTestHalf, ConvCKIgemmFwdBiasActivFused)
         fusePlanDesc, plan_params, conv_config, test_skipped);
 }
 
+#if MIOPEN_BACKEND_HIP
+TEST_P(ConvBiasActivInferTestFloatFusionCompileStep, ConvBiasActivAsm1x1UFloat_testCompile)
+{
+    setEnvironmentVariable("MIOPEN_FIND_ENFORCE", "3");
```

Comment:
Value 3 means that tuning is not performed if the perf-db already contains the performance parameters. For reliability of the test, use "MIOPEN_FIND_ENFORCE=SEARCH_DB_UPDATE".

The full tuning cycle may take thousands of iterations. Use `MIOPEN_DEBUG_TUNING_ITERATIONS_MAX=5` to limit tuning time in tests.